miniapps/performance/ex1.cpp

```diff
@@ -124,6 +124,8 @@ int main(int argc, char *argv[])
       return 3;
    }
 
+   cout << "\nMFEM SIMD width: " << MFEM_SIMD_BYTES << " bytes\n" << endl;
```

Comment:
Should we output `width` `(MFEM_SIMD_BYTES/sizeof(real_t))` or `bytes`? 
I'm more in favor of `width`, which I think is more intuitive for the user.

Reply:
That's true -- I was also thinking about that. I'll switch this to print the width in terms of `doubles`, e.g. `MFEM SIMD width: 4 doubles`.

Reply:
`done`